content/en/docs/concepts/storage/volume-health-monitoring.md

```diff
@@ -0,0 +1,34 @@
+---
+reviewers:
+- jsafrane
+- saad-ali
+- msau42
+- xing-yang
+--
+title: Volume Health Monitoring 
+content_type: concept
+---
+
+<!-- overview -->
+
+This page describes the CSI Volume Health Monitoring feature. CSI Volume Health Monitoring allows CSI Drivers to detect abnormal volume conditions from the underlying storage systems and report them as events on PVCs or Pods.
+
+{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
+
+<!-- body -->
+
+## Volume Health Monitoring
+
+The Volume Health Monitoring feature is part of Kubernetes implementation of Container Storage Interface (CSI). The Volume Health Monitoring feature is impelmeted in two components: External Health Monitor Controller and Kubelet.
+
+The External Health Monitor Controller will be deployed as a sidecar together with the CSI controller driver. It triggers controller RPC to check the health condition of the CSI volumes.
```

Comment:
- I'd write “sidecar container” in case the reader is not familiar with sidecars
- We should give the reader a hint about _how_ that sidecar container is deployed
   - do they define it in a Pod template?
   - does the control plane inject it?

Reply:
Changed to "sidecar container" and included a link to this doc which has a diagram for the sidecar.